lib/searchyll/generator.rb

```diff
@@ -14,6 +14,12 @@ def generate(site)
       # Gather the configuration options
       configuration = Configuration.new(site)
 
+      # Don't do anything if the Elasticsearch URL is missing
+      if configuration.elasticsearch_url.empty?
+        puts "No Elasticsearch URL present, skipping indexing"
```

Comment:
Is there a way to direct the users attention to docs on how to fix?

Reply:
We could change the error message. Another alternative I was considering was simply defaulting to `localhost:9200`, which is what a lot of Elasticsearch clients do anyway. Open to comments on this.